plugin/trino-mongodb/src/main/java/io/trino/plugin/mongodb/MongoSession.java

```diff
@@ -922,7 +922,8 @@ else if (value instanceof Decimal128 decimal128) {
             catch (ArithmeticException e) {
                 return Optional.empty();
             }
-            typeSignature = createDecimalType(decimal.precision(), decimal.scale()).getTypeSignature();
+            int precision = Math.max(decimal.precision(), decimal.scale());
```

Comment:
can you explain why this is sane?

I'm assuming because the Mongo driver doesn't send the precision for values like `0.0x`?

Reply:
I'll add a comment. I explained this in the PR description.